src/NodaTime.Web.Test/NodaTime.Web.Test.csproj

```diff
@@ -14,5 +14,9 @@
     <PackageReference Include="NUnit" Version="3.6.1" />
     <PackageReference Include="NUnitLite" Version="3.6.1" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
(This is https://github.com/Microsoft/vstest/issues/472, and while there's a lot of discussion there about whether this makes sense, it's probably fine to leave it as-is.)

Reply:
Ah, missed this one. I removed it again from all the rest of the projects before merging. Think I'll do so again here.